packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -362,13 +363,69 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
   }
   let currentSeats: CurrentSeats | undefined;
 
-  let usersWithCredentials = eventType.users.map((user) => ({
-    isFixed: !eventType.schedulingType || eventType.schedulingType === SchedulingType.COLLECTIVE,
-    ...user,
-  }));
-  // overwrite if it is a team event & hosts is set, otherwise keep using users.
-  if (eventType.schedulingType && !!eventType.hosts?.length) {
-    usersWithCredentials = eventType.hosts.map(({ isFixed, user }) => ({ isFixed, ...user }));
+  let usersWithCredentials;
+  let teamMember;
+
+  if (eventType.schedulingType === SchedulingType.ROUND_ROBIN && input.bookerEmail) {
+    let crmRoundRobinLeadSkip;
+    // See if CRM app is enabled and skip RR assignment
+    const eventTypeAppMetadata = eventType?.metadata?.apps;
+    for (const appKey in eventTypeAppMetadata) {
+      const app = eventTypeAppMetadata[appKey];
+      if (
+        app.enabled &&
+        app.appCategories &&
+        app.appCategories.some((category) => category === "crm") &&
+        app.roundRobinLeadSkip
+      ) {
+        crmRoundRobinLeadSkip = app;
+        break;
+      }
+    }
+
+    if (crmRoundRobinLeadSkip) {
+      const crmCredential = await prisma.credential.findUnique({
+        where: {
+          id: crmRoundRobinLeadSkip.credentialId,
+        },
+        include: {
+          user: {
+            select: {
+              email: true,
+            },
+          },
+        },
+      });
+      if (crmCredential) {
+        const crm = new CrmManager(crmCredential);
+        const contact = await crm.getContacts(input.bookerEmail, true);
+        if (contact) {
+          // Since this is enabled for round robin event, we iterate through hosts
+          const contactOwner = eventType.hosts.find((host) => host.user.email === contact[0].ownerEmail);
```

Comment:
`contact[0].ownerEmail` does this work for you? it only returns contact id and email for me but not the contact owner email

Reply:
Forgot to pass the param to include the contact host. Should be good now.
[`2c05c8e` (#14556)](https://github.com/calcom/cal.com/pull/14556/commits/2c05c8e3474c42c65ad2b5721fbde4a1d6116966)